doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -1848,188 +1779,6 @@ Response:
 
 + Response 204
 
-# Group Registrations
-
```

Comment:
However, that note will be perfect to be included in the Release Notes associated to thir release candidate.